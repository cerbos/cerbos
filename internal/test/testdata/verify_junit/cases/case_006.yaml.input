-- suite_test.yaml --
---
name: CompanyTestSuite
description: Tests for verifying the company resource policy
principals:
  admin:
    id: admin
    roles:
      - admin
  user:
    id: user
    roles:
      - user
resources:
  company:
    id: xx1
    kind: company

tests:
  - name: Company Test 1
    input:
      principals:
        - admin
      resources:
        - company
      actions:
        - create
    expected:
      - principal: admin
        resource: company
        actions:
          create: EFFECT_ALLOW

  - name: Company Test 2
    input:
      principals:
        - admin
      resources:
        - company
      actions:
        - create
    expected:
      - principal: admin
        resource: company
        actions:
          create: EFFECT_ALLOW
