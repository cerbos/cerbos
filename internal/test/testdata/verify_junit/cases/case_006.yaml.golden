<testsuites errors="0" failures="0" skipped="0" tests="2">
  <testsuite description="Tests for verifying the company resource policy" name="CompanyTestSuite" file="suite_test.yaml" errors="0" failures="0" skipped="0" tests="2">
    <properties></properties>
    <testcase file="suite_test.yaml" classname="admin.company.create" name="Company Test 1">
      <success type="RESULT_PASSED">
        <actual>EFFECT_ALLOW</actual>
        <expected>EFFECT_ALLOW</expected>
      </success>
      <properties>
        <property name="principal">admin</property>
        <property name="resource">company</property>
        <property name="action">create</property>
      </properties>
    </testcase>
    <testcase file="suite_test.yaml" classname="admin.company.create" name="Company Test 2">
      <success type="RESULT_PASSED">
        <actual>EFFECT_ALLOW</actual>
        <expected>EFFECT_ALLOW</expected>
      </success>
      <properties>
        <property name="principal">admin</property>
        <property name="resource">company</property>
        <property name="action">create</property>
      </properties>
    </testcase>
  </testsuite>
</testsuites>